src/components/appheader.main.jsx

```diff
@@ -61,62 +54,96 @@ class AppHeaderMain extends React.Component {
   }
 
   render() {
-    const { hideHeaderNavigation } = this.props;
     const { isOffline } = this.state;
-    return (
-      <div>
-        <header className="app-header navbar navbar -fixed-top navbar-inverse" data-region="appHeader" style={{ display: 'block' }}>
-          <div className="container appheader-container">
-            <div className="back-button-container" style={{ display: 'block' }}>
-              <div>
-                <button type="button" id="header_mobile_back_button" aria-label="Back" className="navbar-back" data-region="backButtonRegion" data-el-label="navigation.back" onClick={AppHeaderMain.goBack}>
```

Comment:
Was the removal of onclick event intentional? The back button in mobile view no longer behaves like before. 

Clicking on this button results in following console error
bootstrap.bundle.min.js:6 Uncaught TypeError: Cannot read property 'setAttribute' of null
    at Object.onLoad (bootstrap.bundle.min.js:6)
    at bootstrap.bundle.min.js:6
    at Array.forEach (<anonymous>)
    at new o (bootstrap.bundle.min.js:6)
    at c.e.toggle (bootstrap.bundle.min.js:6)
    at HTMLButtonElement.<anonymous> (bootstrap.bundle.min.js:6)
    at Function.each (jquery.js:354)
    at jQuery.fn.init.each (jquery.js:189)
    at jQuery.fn.init.c._jQueryInterface [as dropdown] (bootstrap.bundle.min.js:6)
    at HTMLButtonElement.<anonymous> (bootstrap.bundle.min.js:6)

Reply:
No, it was not intentional :) Good catch! The error has to do with the data-toggle. I fixed it both.